angularjs-portal-home/src/main/webapp/my-app/layout/widget/partials/widget-card.html

```diff
@@ -9,14 +9,14 @@
       tooltip-popup-delay="200"></i>
     </div>
     <div class='widget-remove'>
-      <i title="Remove" class="fa fa-times portlet-options" ng-click="widgetCtrl.removePortlet(portlet.nodeId, portlet.title)"></i>
+      <i title="Remove" class="fa fa-times portlet-options" ng-click="widgetCtrl.removePortlet(portlet.nodeId, portlet.title)"><a aria-label="Remove this app" href="#"></a></i>
     </div>
     
     <div class="widget-title">
-      <h4>{{ ::portlet.title }}</h4>
+      <h4 id="appTitle{{::portlet.title}}" aria-labelledby="appTitle{{::portlet.title}}" tabindex="0">{{ ::portlet.title }}</h4>
     </div>
   </div>
-  
+  <sub hidden="" id="goToApps">go to</sub>
```

Comment:
Yes, 
![widgecard](https://cloud.githubusercontent.com/assets/10791982/8786733/7b45642a-2ef5-11e5-8b6c-45f5182b85ac.PNG)
so is that ok if I just change the id and leave <sub> tag there like this?